lib/findbugs_translate_checkstyle_format/translate.rb

```diff
@@ -22,7 +22,13 @@ def trans(xml)
 
       bug_instances = [bug_instances] if bug_instances.is_a?(Hash)
       bug_instances.each do |bug_instance|
-        source_lines = bug_instance['SourceLine']
+        source_lines = if bug_instance.has_key?('SourceLine')
+          bug_instance['SourceLine']
+        elsif bug_instance['Class'].is_a?(Array)
+          bug_instance['Class'].map{|classes| classes['SourceLine']}
+        else
+          bug_instance['Class']['SourceLine']
+        end
```

Comment:
__[rubocop]__
* `end` at 31, 8 is not aligned with `if` at 25, 23.

<!-- Comment by [SideCI](https://sideci.com) -->